wallet-api/src/Ledger/Value.hs

```diff
@@ -52,6 +54,11 @@ currencySymbol = $$(TH.currencySymbol)
 tokenName :: P.ByteString -> TokenName
 tokenName = $$(TH.tokenName)
 
+-- | Convert a Haskell 'String' to a PLC 'TokenName' using
+--   the Char8 encoding (see 'Data.ByteString.Char8').
+mkTokenName :: String -> TokenName
+mkTokenName = tokenName . P.SizedByteString . C8.pack
```

Comment:
It seems bad that this isn't the same as the `IsString` instance!

Reply:
It is now.